pkg/providers/ovirt/mapper/mapper.go

```diff
@@ -391,6 +393,17 @@ func (o *OvirtMapper) mapNetworkType(mapping v2vv1alpha1.ResourceMappingItem, ku
 
 // ResolveVMName resolves the target VM name
 func (o *OvirtMapper) ResolveVMName(targetVMName *string) *string {
+	vmNameBase := o.resolveVMNameBase(targetVMName)
+	if vmNameBase == nil {
+		return nil
+	}
+	// VM name is put in label values and has to be shorter than regular k8s name
+	// https://bugzilla.redhat.com/show_bug.cgi?id=1857165
```

Comment:
No need to add `show_bug.cgi?id=` in the url

Reply:
Done